metagraph/src/cli/transform_annotation.cpp

```diff
@@ -46,6 +46,16 @@ int transform_annotation(Config *config) {
 
     const auto &files = config->fnames;
 
+    if (config->anno_type == Config::RowDiff && !files.size()) {
+        logger->trace("Passed no input annotations to transform. Prepare to RowDiff transform...");
```

Comment:
This message is kind of unintuitive, why not have it say something like "Preparing to RowDiff transform in output directory" or something like that"